README.md

```diff
@@ -1,4 +0,0 @@
-# Basecode Repository
```

Comment:
Please make a new README, instead of just deleting it flat out, it's good for visiting people.